src/frontends/tensorflow/src/helper_transforms/block_lstm_replacer.cpp

```diff
@@ -0,0 +1,168 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "helper_transforms/block_lstm_replacer.hpp"
+
+#include <memory>
+#include <vector>
+
+#include "helper_ops/block_lstm.hpp"
+#include "ngraph/rt_info.hpp"
+#include "openvino/opsets/opset8.hpp"
+#include "openvino/pass/pattern/matcher.hpp"
+#include "openvino/pass/pattern/op/or.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "transformations/utils/utils.hpp"
+
+using namespace std;
+using namespace ov::pass;
+using namespace ov::opset8;
+using namespace ov::frontend::tensorflow;
+
+ov::frontend::tensorflow::pass::BlockLSTMToLSTMSequenceOneOutput::BlockLSTMToLSTMSequenceOneOutput() {
+    auto block_lstm = ov::pass::pattern::wrap_type<BlockLSTM>();
+
+    matcher_pass_callback callback = [=](ov::pass::pattern::Matcher& m) {
```

Comment:
you already have usings in this file, so please remove ov:: , ov::pass:: and so on

Reply:
Done